AmplifyPlugins/Core/AWSPluginsCore/Model/Support/SelectionSetField.swift

```diff
@@ -0,0 +1,34 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+
+public class SelectionSetField {
```

Comment:
Does this need class semantics (reference, inheritance)

Reply:
no, updating :D